### PR TITLE
LPS-192308 | master

### PR DESCRIPTION
**Pull request by brookedalton**
https://liferay.atlassian.net/browse/LPS-192308

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SearchBarSuggestions.testcase [hunk 1]
```diff
@@ -741,6 +741,43 @@ definition {
...
+
+		Type(
+			locator1 = "Search#EMBEDDED_SEARCH_BAR",
+			value1 = "Title");
+
+		SearchBarSuggestions.viewSuggestions(
+			searchAssetDescription = "descripción",
+			searchAssetTitle = "Título",
+			suggestionContributorDisplayGroupName = "Sugerencias");
+
```
Comment (joshchong) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/search/searchportlet/SearchBarSuggestions.testcase:763]: can you add a negative assertion here similar to what you did when searching in german locale?
```
		AssertElementNotPresent(
			key_searchAssetDescription = "description",
			key_searchAssetTitle = "Titel",
			key_suggestionContributorDisplayGroupName = "Vorschläge",
			locator1 = "SearchBarSuggestions#RESULTS_LIST_ITEM_DESCRIPTION");

**Review by joshchong — CHANGES_REQUESTED** [review]
you can forward the test after making the small change above

**Comment by brookedalton** [comment]
ci:test:sf

**Comment by brookedalton** [comment]
ci:forward

**Pull request closed without merging**